### ISSUE TITLE
manpages wrong

### ISSUE THREAD

**Issue by shagu** [esno/test#22]
The manpages seem to be out of date. When I type:

```
$ man test
```
I get manpages that seem to be unrelated to this project. Please fix.

PS: I'm using Linux 2000 Professional.

**Comment by esno** [comment]
yes that's why we're in a lawsuit with KB ad MB. I'm sure they just pressed fork and published all our work.
